extensions/visual-studio-code-snippet-manager/src/components/Editor.tsx

```diff
@@ -0,0 +1,72 @@
+import { ActionPanel, Action, Form, useNavigation, showToast, Toast, popToRoot } from "@raycast/api";
+import { randomUUID } from "crypto";
+
+import fs from "fs";
+import { languages, path } from "../data";
+import Search from "./Search";
+
+const Editor = (props?: Item) => {
+  const { push } = useNavigation();
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm
+            onSubmit={(v) => {
+              if (v.title === "" || v.prefix === "" || v.body === "") {
+                showToast({
+                  title: "error",
+                  message: "title, prefix and body are required",
+                  style: Toast.Style.Failure,
+                });
+              } else {
+                const data = {
+                  title: v.title,
```

Comment:
As far as I know, snippets don't have a `title` property, but instead the title is inferred from the key.
```
	// Place your snippets for javascript here. Each snippet is defined under a snippet name and has a prefix, body and 
	// description. The prefix is what is used to trigger the snippet and the body will be expanded and inserted. Possible variables are:
	// $1, $2 for tab stops, $0 for the final cursor position, and ${1:label}, ${2:another} for placeholders. Placeholders with the 
	// same ids are connected.
	// Example:
	// "Print to console": {
	// 	"prefix": "log",
	// 	"body": [
	// 		"console.log('$1');",
	// 		"$2"
	// 	],
	// 	"description": "Log output to console"
	// }
```

<img width="649" alt="image" src="https://user-images.githubusercontent.com/1155589/165081337-7729ea00-0e2a-48a7-9d76-ae54f21b1b50.png">